mllib/src/main/scala/org/apache/spark/ml/clustering/LDA.scala

```diff
@@ -562,6 +618,43 @@ class DistributedLDAModel private[ml] (
    */
   @Since("1.6.0")
   lazy val logPrior: Double = oldDistributedModel.logPrior
+
+  @Since("1.6.0")
+  override def write: MLWriter = new DistributedLDAModel.DistributedLDAModelWriter(this)
```

Comment:
Sorry, but do you mean we can change it to `DistributedLDAModel.DistributedWriter(this)` ?